src/javascript/app/pages/cashier/cashier.js

```diff
@@ -30,6 +30,7 @@ const Cashier = (() => {
             $toggler.toggleClass('open');
         });
         showCashierNote();
+        showP2PNote();
```

Comment:
this should be onLoad not in showcontent. we're not showing it in paymentmethods page so it doesn't need to be here.